src/etos_client/lib/graphql_queries.py

```diff
@@ -44,6 +44,20 @@
 }
 """
 
+ACTIVITY_CANCELED = """
+{
+  activityCanceled(search: "{'links.target': '%s'}") {
```

Comment:
Might not be necessary, but adding links.type=ACTIVITY_EXECUTION might make this more explicit

Reply:
Fixed